src/components/notification/Notifcation.js

```diff
@@ -17,7 +17,7 @@ class Notification extends Component {
       error: false,
     };
     this.socket = io('https://devrepublic-bn-backend.herokuapp.com', {
```

Comment:
As the PO requested, we shouldn't use a hardcoded URL. But as I can see, here we still have the backend URL. Can you fix this and elsewhere that you used the same?